core/src/main/java/com/homeaway/streamplatform/streamregistry/dto/AvroToJsonDTO.java

```diff
@@ -88,9 +80,38 @@ public static Stream convertAvroToJson(AvroStream avroStream) {
                 .topicConfig(avroStream.getTopicConfig())
                 .partitions(avroStream.getPartitions())
                 .replicationFactor(avroStream.getReplicationFactor())
+                .alertConfigList(convertAlertListToJson(avroStream.getAlertConfigList()))
+                .connectorConfigList(convertConnectionConfigListToJson(avroStream.getConnectorConfigList()))
                 .build();
     }
 
+    public static List<ConnectorConfig> convertConnectionConfigListToJson(List<com.homeaway.digitalplatform.streamregistry.Connector> list) {
+        if (list == null) {
+            return new ArrayList<>();
+        }
+        ArrayList<ConnectorConfig> retList = new ArrayList<>();
+        for (com.homeaway.digitalplatform.streamregistry.Connector elem : list) {
+            if (elem != null) {
+                retList.add(new ConnectorConfig(elem.getName(), elem.getType(), elem.getProperties()));
+            }
+        }
+        return retList;
+
+    }
+
+    public static List<AlertConfig> convertAlertListToJson(List<com.homeaway.digitalplatform.streamregistry.Alert> list) {
+        if (list == null) {
+            return new ArrayList<>();
+        }
+        ArrayList<AlertConfig> retList = new ArrayList<>();
+        for (com.homeaway.digitalplatform.streamregistry.Alert elem : list) {
+            if (elem != null) {
+                retList.add(new AlertConfig(elem.getType(), elem.getDestination()));
+            }
+        }
+        return retList;
```

Comment:
I believe most of this could be collapsed 
```java
return list.filter(Objects::nonNull).map(e -> new AlertConfig(...)).collect(Collectors.toList());
```